makefile

```diff
@@ -65,6 +67,13 @@ serpent_only:
 	grep oops build/test_summary.txt && exit 13 || exit 0
         # make end_testing
 
+cgra_pnr_only:
+	# make start_testing
+	@echo 'CGRA PnR' 		>> build/test_summary.txt
+	make cgra_pnr_tests || (echo oops cgra_pnr failed | tee -a build/test_summary.txt)
+	grep oops build/test_summary.txt && exit 13 || exit 0
+	# make end_testing
```

Comment:
Same with these `make` comments, do we need to include them?

Reply:
> make cgra_pnr_tests || (echo oops cgra_pnr failed | tee -a build/test_summary.txt)
> grep oops build/test_summary.txt && exit 13 || exit 0

The "oops" is supposed to be a way of summarizing failures at the end of the travis log in the case where
* travis does not immediately fail on every error, and
* the test generating the "oops" is followed by other tests that might otherwise obscure the failure (in which case the "grep oops" should be the last thing in the travis script, and not here in the makefile).

My preference would be to move the "grep oops" as last thing in the travis script

Reply:
Moving the grep oops and exit code to the end of the travis script makes sense to me. If I understand this correctly, including it here means we'll have an early exit on a failure, so we may not see other tests that will also fail (which could be useful in debugging the failure)

Reply:
Yes that's right.